dev-requirements-py3.txt

```diff
@@ -1,7 +1,7 @@
 flake8==3.7.9
 bandit==1.6.2
 demisto-py==2.0.9
-demisto-sdk==0.3.6
+demisto-sdk==0.3.8
```

Comment:
@bakatzir can you try to revert it to 0.3.7?

Reply:
seems not related @Bars92